_episodes/34.md

```diff
@@ -0,0 +1,298 @@
+hive.s3.endpoint=http://minio:9000
+hive.s3.aws-access-key=minio
+hive.s3.aws-secret-key=minio123
+hive.s3.path-style-access=true
+delta.enable-non-concurrent-writes=true
+```
+
+Once everything is up and running we can start playing.
+
+Verify that the catalog is available:
+
+```sql
+SHOW CATALOGS;
+```
+
+Check if there are any schemas:
+
+```sql
+SHOW SCHEMAS FROM delta;
+```
+
+Lets create a new schema:
+
+```sql
+CREATE SCHEMA delta.myschema WITH (location='s3a://claudiustestbucket/myschema');
+```
+
+Create a table, insert some records, and then verify:
+
+```sql
+CREATE TABLE delta.myschema.mytable (name varchar, id integer);
+INSERT INTO delta.myschema.mytable VALUES ( 'John', 1), ('Jane', 2);
+SELECT * FROM delta.myschema.mytable;
+```
+
+Run a query to get more data and insert it into a new table:
+
+```sql
+CREATE TABLE delta.myschema.myothertable AS
+  SELECT * FROM delta.myschema.mytable;
+
+SELECT * FROM delta.myschema.myothertable ;
+```
+
+Now for some data manipulation:
+
+```sql
+UPDATE delta.myschema.myothertable set name='Jonathan' where id=1;
+SELECT * FROM delta.myschema.myothertable ;
+DELETE FROM delta.myschema.myothertable where id=2;
+SELECT * FROM delta.myschema.myothertable ;
+```
+
+And finally, lets clean up:
+
+```sql
+ALTER TABLE delta.myschema.mytable EXECUTE optimize(file_size_threshold => '10MB');
+ANALYZE delta.myschema.myothertable;
+DROP TABLE delta.myschema.myothertable ;
+DROP TABLE delta.myschema.mytable ;
+DROP SCHEMA delta.myschema;
+```
+
+As you can see with Trino and Delta Lake you get full create, read, update, and
+delete operations on your lake house.
+
+## Question of the episode: How do I secure the connection from a Trino cluster to the data source
+
+Since we talked about connectors earlier, you already know that the
+configuration for accessing a data source is assembled to create a catalog. This
+approach uses a properties file in `etc/catalog`. For example, let's look at the
+recently changed [SQL Server connector
+documentation](../docs/current/connector/sqlserver.html):
+
+```properties
+connector.name=sqlserver
+connection-url=jdbc:sqlserver://<host>:<port>;database=<database>;encrypt=false
+connection-user=root
+connection-password=secret
+```
+
+The connector uses username and password authentication. It connects using the
+JDBC driver, which in turn defaults to TLS enabled. A number of other connectors
+also use JDBC drivers with username and password authentication, but details
+vary a lot. However for all of them you can use [secrets support in
+Trino](../docs/current/security/secrets.html) to use environment variable
+references instead of hardcoding passwords.
+
+When it comes to other connectors the details of securing a connection vary even
+more. Ultimately the answer to how to secure the connection, and if that is even
+possible, is the usual "It depends". Luckily you can check the documentation for
+each connector to find out more and ping us on slack if you need more help.
```

Comment:
link "connector" to the connector overview page

Reply:
Slack should be uppercase, and would suggest specifying a channel (so as not to encourage DMing Brian on his break 😉  )